src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -237,11 +238,17 @@ def nlcd_streams(sjs_result):
                          for count in (ag_count,
                                        low_urban_count,
                                        med_high_urban_count))
+    lu_streams = [0.0] * 16
+    for k, v in result.iteritems():
+        lu = get_lu_index(k)
+        if lu is not None:
+            lu_streams[lu] += float(v) / total
 
     return {
         'ag_stream_pct': ag,
         'low_urban_stream_pct': low,
-        'med_high_urban_stream_pct': med_high
+        'med_high_urban_stream_pct': med_high,
+        'lu_streams': lu_streams
```

Comment:
Hmm, perhaps we should call it `lu_stream_pct` to be consistent with the other keys, and since it does only store the percentage value, not the actual number of streams.